test/e2e/tools/eks-anywhere-test-tool/pkg/filewriter/writer.go

```diff
@@ -60,6 +62,11 @@ func (t *writer) Dir() string {
 // This method writes the e2e test artifacts from S3 to files in a directory named after the e2e test name.
 func (t *writer) WriteTestArtifactsS3ToFile(key string, data []byte) error {
 	i := strings.LastIndex(key, "/Test")
+	if i == -1 {
+		logger.Info("Failed writing object to file", "key", key)
+		return nil
```

Comment:
why log and return nil instead of returning err?

Reply:
Because we want to fetch the other ones after?